src/components/pages/NextStepsSuccess.js

```diff
@@ -32,59 +77,64 @@ const NextStepsSuccess = ({ quiz }) => {
       <Box mb={2}>
         <Typography variant="h1">You're All Set!</Typography>
       </Box>
-      <Box mb={2}>
+      <Box mb={4}>
         <Typography variant="body1">
-          If you don't see an email from compensation@freefrom.org soon, please
-          check your Spam folder.
+          Make sure to add compass@freefrom.org to your address book so you receive our emails.
+          If you don't see an email from us in the next day, please check your Spam folder.
         </Typography>
       </Box>
-      <Box mb={2} mt={4}>
-        <Typography variant="body1">
-          In the meantime, we have a few quick questions about how we can
-          improve this tool. Sharing your feedback will take less than one
-          minute.
-        </Typography>
-        <Box mt={2}>
-          <Grid container item justify="center" alignItems="center">
-            <Button
-              color="primary"
-              variant="contained"
-              href="https://www.surveymonkey.com/r/MYYY7L5"
-              target="_blank"
-            >
-              Share Feedback
-            </Button>
-          </Grid>
+
+      <Divider light={true} />
+
+      <Box className={classes.buttonBox} display="flex" alignItems="flex-start" justifyContent="space-between" paddingBottom={4} paddingTop={4} >
+        <Box maxWidth={500}>
+          <Typography variant="body1">
+            We have a few quick questions about how we can
+            improve this tool. Sharing your feedback will take less than one
+            minute.
+          </Typography>
         </Box>
+        <Button
+          className={classes.actionButton}
+          color="primary"
+          variant="contained"
+          href="https://www.surveymonkey.com/r/MYYY7L5"
+          target="_blank"
```

Comment:
Google Lighthouse caught this security issue:
```suggestion
          target="_blank"
          rel="noopener noreferrer"
```
>In general, when you use target="_blank", always add rel="noopener" or rel="noreferrer"

https://web.dev/external-anchors-use-rel-noopener/

Reply:
Beautiful, thank you 👌